charts/kusion/README.md.gotmpl

```diff
@@ -57,16 +63,16 @@ helm install kusion-release ./kusion
 First, you need to base64 encode your kubeconfig files.
 
```

Comment:
The new command using 'cat' and 'tr' is less efficient than the previous version using 'base64 -w 0'. Consider reverting to the original approach for better performance.